src/MessageDispatch.KurrentDB/KurrentDbSubscriber.cs

```diff
@@ -73,13 +83,38 @@ private KurrentDbSubscriber(
         KurrentDBClient kurrentDbClient,
         IDispatcher<ResolvedEvent> dispatcher,
         string streamName,
-        ILogger logger)
-        => Init(kurrentDbClient, dispatcher, streamName, logger, liveOnly: true);
-
+        ILogger logger)
+    {
+        SetupCallbackTimer();
+
+        Init(kurrentDbClient, dispatcher, streamName, logger, liveOnly: true);
+    }
+
+    /// <summary>
+    /// Creates a callback timer to ping KurrentDB every 60s to stop it from going idle
+    /// </summary>
+    private void SetupCallbackTimer()
+    {
+        //Create the timer and set it for a sixty second interval
+        _timer = new Timer();
+        _timer.Interval = 60000;
+
+        _timer.Elapsed += CallbackEvent;
+
+        _timer.AutoReset = true;
+
+        _timer.Enabled = true;
+    }
+
+    private void CallbackEvent(object source, ElapsedEventArgs e)
+    {
+        _kurrentDbClient.ReadStreamAsync(Direction.Backwards, _streamName, StreamPosition.End);

```

Comment:
Might make more sense to read a single event here, I'm not sure what it's doing here as I can't see a number of events specified.